app/config/locale/translations/en.json

```diff
@@ -43,6 +43,12 @@
     "emails.invitation.footer": "If you are not interested, you can ignore this message.",
     "emails.invitation.thanks": "Thanks",
     "emails.invitation.signature": "{{project}} team",
+    "emails.certificate.subject": "Certificate failure for %s",
```

Comment:
Hmm, I think we had a script to generate translations for all languages, from EN. using AI. We could run it.

Reply:
https://github.com/appwrite/appwrite/pull/7442

We might need to check with CHristy for thisPR to be merged beforehand

Reply:
Do you know where this script is?